README.md

```diff
@@ -132,6 +144,11 @@ Get a data set
 ```
 zeg get dataset [dataset id] --project [Project Id] --token [API token]
 ```
+The dataset id can be found by by running the following on the cli:
+
+zeg get collection <collection id> --project <project id>
+
+From here we will be able to see the upload_dataset_id which can be thought of as the data input funnel (whereas dataset_id is the data output pipe that the viewer pulls from).
```

Comment:
```From here `upload_dataset_id` can be obtained. This identifies the dataset that represents the data as it was uploaded. Whereas `dataset_id`  identifies the dataset processed delivered to the viewer.```

Reply:
thanks for the feedback @tarqu1n ! Will implement now 👍